lib/index.js

```diff
@@ -64,7 +65,7 @@ function TradeOfferManager(options) {
 	}
 }
 
-TradeOfferManager.prototype.setCookies = function(cookies, callback) {
+TradeOfferManager.prototype.setCookies = (cookies, callback) => {
```

Comment:
I prefer to not do arrow functions for member functions, as `this` and `arguments` can be needed. In fact, what *is* the value of `this` when you do this?